inetum-realjobs-backend/src/test/resources/application.properties

```diff
@@ -3,3 +3,8 @@ spring.datasource.username=test
 spring.datasource.password=test
 spring.jpa.hibernate.ddl-auto=create-drop
 spring.sql.init.mode=never
+spring.mail.host=localhost
+spring.mail.port=3025
+spring.mail.username=mail
+spring.mail.password=test
+spring.mail.test-connection=true
```

Comment:
Does this meant to be pushed on git?

Reply:
Yes, these are for GreenMail, which runs alongside the tests.